lib/screens/permiso_covid_screen.dart

```diff
@@ -0,0 +1,157 @@
+import 'package:flutter/material.dart';
+import 'package:mi_utem/widgets/custom_app_bar.dart';
+import 'package:qr_flutter/qr_flutter.dart';
+
+class PermisoCovidScreen extends StatelessWidget {
+  const PermisoCovidScreen({Key? key}) : super(key: key);
```

Comment:
Aquí tendrías que recibir el `PermisoCovid` como `required this.permiso` parecido a como lo haces en `QRCard`, y luego llamarlo en la navegación como Get.to(PermisoCovidScreen(permiso: permiso))